gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
+    -----------
+    obs_table:  `~gammapy.data.ObservationTable`
+        Observation index table
+    target_pos: `~astropy.coordinates.SkyCoord`
+        Target position
+    """
+
+    def __init__(self, obs_table, target_pos=None):
+        self.obs_table = obs_table
+        self.target_pos = target_pos
+
+    def get_offset(self):
+        """
+        Compute offsets of the different observations
+        
+        Returns:
+        --------
+        vector: `~numpy.array`
+            Offsets
+        """
+        pnt_coord = SkyCoord(np.array(self.obs_table['RA_PNT']),
+                             np.array(self.obs_table['DEC_PNT']),
+                             unit='deg')
+
+        return pnt_coord.separation(self.target_pos)
+    
+    def plot_zenith_distribution(self, ax=None, bins=100, range=(0,100.)):
```

Comment:
I would prefer if you pass in the binning spec like this:
```
def plot_something(bins=None):
    zen = self.obs_table['ZEN_PNT']
    if bins is None:
        bins = np.linspace(0, zen.max() + 5, 10)
```
This has advantages:
1. The caller can pass in an arbitrary binning.
2. There's just one arg, not two
3. You have more flexibility to compute the default via code

It's hard to set good defaults for plots.
Here probably 0 ... 90 would be acceptable as well and it's pretty easy.
In other plots we have hard-coded values that are appropriate for HESS, i.e. offset 0 ... 2.5.

I'm not sure how to do this better ... I do think it's important to try to put defaults so that it's easy to use for common cases.